<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-M0OJOK9" ignored="0" name="Test" tests="15" failures="3" timestamp="07 Jan 2022 00:35:41 GMT" time="148.428" errors="0">
  <testcase name="TC01" time="4.164" classname="Railway.TC01"/>
  <testcase name="TC02" time="3.588" classname="Railway.TC02"/>
  <testcase name="TC03" time="4.432" classname="Railway.TC03"/>
  <testcase name="TC04" time="5.271" classname="Railway.TC04"/>
  <testcase name="TC05" time="6.631" classname="Railway.TC05">
    <failure type="java.lang.AssertionError" message="System doesn&amp;apos;t show message when user enters wrong password several times expected [You have used 4 out of 5 login attempts. After all 5 have been used, you will be unable to login for 15 minutes.] but found [Invalid username or password. Please try again.]">
      <![CDATA[java.lang.AssertionError: System doesn't show message when user enters wrong password several times expected [You have used 4 out of 5 login attempts. After all 5 have been used, you will be unable to login for 15 minutes.] but found [Invalid username or password. Please try again.]
at Railway.TC05.TC05(TC05.java:33)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- TC05 -->
  <testcase name="TC06" time="6.764" classname="Railway.TC06"/>
  <testcase name="TC07" time="4.026" classname="Railway.TC07"/>
  <testcase name="TC08" time="3.518" classname="Railway.TC08">
    <failure type="java.lang.AssertionError" message="User can login with un-active account expected [Invalid username or password. Please try again.] but found []">
      <![CDATA[java.lang.AssertionError: User can login with un-active account expected [Invalid username or password. Please try again.] but found []
at Railway.TC08.TC08(TC08.java:43)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- TC08 -->
  <testcase name="TC09" time="5.741" classname="Railway.TC09"/>
  <testcase name="TC10" time="3.214" classname="Railway.TC10"/>
  <testcase name="TC11" time="3.831" classname="Railway.TC11"/>
  <testcase name="TC14" time="6.963" classname="Railway.TC14"/>
  <testcase name="TC15" time="5.996" classname="Railway.TC15">
    <failure type="java.lang.AssertionError" message="Table Title is not displayed correctly expected [Ticket price from Đà Nẵng to Sài Gòn] but found [Ticket price from Sài Gòn to Phan Thiết]">
      <![CDATA[java.lang.AssertionError: Table Title is not displayed correctly expected [Ticket price from Đà Nẵng to Sài Gòn] but found [Ticket price from Sài Gòn to Phan Thiết]
at Railway.TC15.TC15(TC15.java:78)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- TC15 -->
  <testcase name="TC16" time="9.208" classname="Railway.TC16"/>
  <testcase name="TC17" time="14.179" classname="Railway.TC17"/>
</testsuite> <!-- Test -->
